### PR TITLE
Upstream: Pull in changes from Hugo v0.79.0

### PR DESCRIPTION
**Pull request by felicianotech**

### CODE REVIEW

File: examples/blog/layouts/partials/meta.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
 
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="{{ .Description }}">
-    <meta name="author" content="A Hugo User"> <!-- This should be modified to be your name, if you want to include this information -->
\ No newline at end of file
+    <meta name="author" content="A Hugo User"> <!-- This should be modified to be your name, if you want to include this information -->
```
Comment (chayev) [inline examples/blog/layouts/partials/meta.html:5]: We should update this to say Gotham instead.

File: src/hugolib/config.go [hunk 2]
```diff
@@ -166,45 +168,59 @@ func LoadConfig(d ConfigSourceDescriptor, doWithConfig ...func(cfg config.Provid
...
+	const delim = "__env__delim"
+
 	// Apply environment overrides
 	if len(d.Environ) > 0 {
-		// Extract all that start with the HUGO_ prefix
-		const hugoEnvPrefix = "HUGO_"
-		var hugoEnv []string
+		// Extract all that start with the HUGO prefix.
+		// The delimiter is the following rune, usually "_".
+		const hugoEnvPrefix = "HUGO"
```
Comment (chayev) [inline src/hugolib/config.go:177]: Does this need to be changed as well?

**Review by chayev — COMMENTED** [review]
Added a few comments. All looks good. 

The comments might need to be addressed outside of this PR, so no explicitly requesting changes.

File: src/hugolib/config.go (hunk 2 above)
Comment (felicianotech) [reply src/hugolib/config.go:177]: Not in v1. That would break backwards compatibility.

File: examples/blog/layouts/partials/meta.html (hunk 1 above)
Comment (felicianotech) [reply examples/blog/layouts/partials/meta.html:5]: Why?

**Comment by felicianotech** [comment]
> so no explicitly requesting changes.

@chayev But you didn't approve either so it's still waiting for an approval.

File: examples/blog/layouts/partials/meta.html (hunk 1 above)
Comment (chayev) [reply examples/blog/layouts/partials/meta.html:5]: Our users would see "A Hugo User" there. Your call, we should review this outside of this PR, as this has been like that before this update.

**Review by chayev — APPROVED** [review]
LG!

File: examples/blog/layouts/partials/meta.html (hunk 1 above)
Comment (felicianotech) [reply examples/blog/layouts/partials/meta.html:5]: I opened this back up. I question if users would even see this. I thought it was more of like a test, though it's not named that. Now I'm not sure.

**Pull request merged**